src/components/Button.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Button(props) {
+  return (
+    <button type="button" className="square">
+      {props.name}
+    </button>
+  );
+}
+
+Button.propTypes = {
+  name: PropTypes.string,
+};
```

Comment:
to avoid the prop type eslint error you can use `name: PropTypes.string.required`